### ISSUE TITLE
Add "open related" to the icon bar

### ISSUE THREAD

**Issue by m1ga** [appcelerator/atom-appcelerator-titanium#174]
Thinking about to add "open related" to the icons bar, so you can quickly open the other files by a mouse click (calling the existing function). 
Not sure about the best implementation:
* Drop Down where you can select "controller, view, style, all other" and a "go" button
* Button with icons for each of the above options
* different approach

any ideas?

**Comment by ewanharris** [comment]
I like the idea of making the open related functionality more accessible.

The way we exposed this in vscode was to add to the context menu shown on right click when clicking on an open editor pane (I think I also intended to add this to right clicking a file in the file explorer but vscode didn't support that).

It looks like atom does support this via the [ContextMenuManager](https://flight-manual.atom.io/api/v1.39.1/ContextMenuManager/).

How does that sound as a different approach to exposing this? I'm wary to continue adding to the toolbar as it could eventually get rather cluttered.
